oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/IndexController.java

```diff
@@ -60,12 +80,16 @@ public String generateDocId(String logicTableName, String originalID) {
      * Check the mode of the Model definition.
      */
     public boolean isMetricModel(Model model) {
-        return StringUtil.isNotBlank(model.getAggregationFunctionName());
+        return Metrics.class.isAssignableFrom(model.getAClass());
+    }
+
+    public boolean isFunctionMetricModel(Model model) {
```

Comment:
```suggestion
    public boolean isFunctionMetric(Model model) {
```

Model has been in the parameter list, we could short the method name.